inc/osvr/Client/CreateContext.h

```diff
@@ -35,12 +35,17 @@
 // Standard includes
 // - none
 
+class vrpn_ConnectionPtr;
+
 namespace osvr {
 namespace client {
 
     OSVR_CLIENT_EXPORT common::ClientContext *
     createContext(const char appId[], const char host[] = "localhost");
 
+    OSVR_CLIENT_EXPORT common::ClientContext *
+    createAnalysisClientContext(const char appId[], const char host[],
+                                vrpn_ConnectionPtr const& conn);
```

Comment:
Different subclass, is created with an already-existing VRPN connection (the server connection generally). Doesn't spin on startup to wait for a connection or path tree. May in the future share resources (path tree) if it makes sense. Also will probably be able to create client interface objects (ugh that name) with relative paths soon, as a way of getting its input.